doc/roadmap.rst

```diff
@@ -56,38 +56,39 @@ The list is numbered not as an indication of the order of priority, but to
 make referring to specific points easier. Please add new entries only at the
 bottom.
 
-#. Everything in Scikit-learn should conform to our API contract
-
-   * `Pipeline <pipeline.Pipeline>` and `FeatureUnion` modify their input
-     parameters in fit. Fixing this requires making sure we have a good
-     grasp of their use cases to make sure all current functionality is
-     maintained. :issue:`8157` :issue:`7382`
 
 #. Improved handling of Pandas DataFrames and SparseDataFrames
 
    * document current handling
    * column reordering issue :issue:`7242`
    * avoiding unnecessary conversion to ndarray :issue:`12147`
    * returning DataFrames from transformers :issue:`5523`
-   * getting DataFrames from dataset loaders :issue:`10733`, :issue:`13902`
+   * getting DataFrames from dataset loaders :issue:`10733`
    * Sparse currently not considered :issue:`12800`
 
 #. Improved handling of categorical features
 
    * Tree-based models should be able to handle both continuous and categorical
-     features :issue:`4899`
-   * In dataset loaders :issue:`13902`
+     features :issue:`12866`
    * As generic transformers to be used with ColumnTransforms (e.g. ordinal
      encoding supervised by correlation with target variable) :issue:`5853`,
      :issue:`11805`
+   * Handling mixtures of categorical and continuous variables
 
 #. Improved handling of missing data
 
-   * Making sure meta-estimators are lenient towards missing data
-   * Non-trivial imputers :issue:`11977`, :issue:`12852`
-   * Learners directly handling missing data :issue:`13911`
+   * Making sure meta-estimators are lenient towards missing data,
+     :issue:`15319`
+   * Non-trivial imputers
```

Comment:
I guess we can even remove it now unless there are other open issues/PR

Reply:
The iterative imputer still has a blocker issue.